lib/bap_c/bap_c_abi.ml

```diff
@@ -182,6 +182,11 @@ let stage2 stage1 = object
     else prog
 end
 
+module StringMap = Map.Make(String)
```

Comment:
The `Map` module is already available in the `String` module, vid., `String.Map`. But I think that using `Hashtbl` here is much more natural, e.g.,

```
let registry = Hashtbl.create (module String)
...
```

Could you please switch to Hashtbl here?